osf/modm_compat.py

```diff
@@ -124,6 +124,9 @@ def argument(self):
     def op(self):
         if self.__val is None:
             return 'isnull'
+        if 'iexact' in self.key:
```

Comment:
Yeah, this isn't ideal. Where is this relevant? The only places I see are when we're filtering tags, e.g. https://github.com/CenterForOpenScience/osf.io/blob/2c4b8255df41f93374081f4019a6c030676dfecf/api/base/filters.py#L453